src/trx/game/output/mesh_batcher/batcher.c

```diff
@@ -416,7 +416,7 @@ static void M_RenderPass(
 {
     MESH_BATCHER *const batcher = source->priv;
 
-    if (pass == SCENE_PASS_OPAQUE || pass == SCENE_PASS_SKYBOX) {
+    if (pass == SCENE_PASS_OPAQUE) {
         M_OpaquePass(batcher, pass);
```

Comment:
No longer needed to pass the `pass` argument - all opaque steps can be simplified in this module now